Formula/git-flow-cjs.rb

```diff
@@ -0,0 +1,49 @@
+class GitFlowCjs < Formula
+  desc "AVH edition of git-flow"
+  homepage "https://github.com/CJ-Systems/gitflow-cjs"
+  license "BSD-2-Clause"
+
+  stable do
+    url "https://github.com/CJ-Systems/gitflow-cjs/archive/refs/tags/v2.2.0.tar.gz"
+    sha256 "e545afd143ce2ac61d4fb1a6d68046eee9b827363009c5aa6db0bf4267a57f08"
+
+    resource "completion" do
+      url "https://github.com/CJ-Systems/git-flow-completion/archive/0.6.0.tar.gz"
+      sha256 "b1b78b785aa2c06f81cc29fcf03a7dfc451ad482de67ca0d89cdb0f941f5594b"
+    end
+  end
+
+  head do
+    url "https://github.com/CJ-Systems/gitflow-cjs.git", branch: "develop"
+
+    resource "completion" do
+      url "https://github.com/CJ-Systems/git-flow-completion.git", branch: "develop"
+    end
+  end
+
+  uses_from_macos "gnu-getopt"
```

Comment:
I needed to pull down `gnu-getopt` from `brew` to get it working, may need to `depends_on "gnu-getopt"` instead.